### PR TITLE
data: qadatta

### PR DESCRIPTION
**Pull request by qadatta**

### CODE REVIEW

File: qadatta.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  "links": [
+    {
+      "name": "Twitter",
+      "url": "https://twitter.com/DattatryaLMore",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "LinkedIn",
+      "url": "https://www.linkedin.com/in/dattatrya-more-94b57513/",
+      "icon": "FaLinkedIn"
```
Comment (khushalarora) [inline qadatta.json:14]: Please replace it with the value  **' FaLinkedin '**  :+1: 
```suggestion
      "icon": "FaLinkedin"
```

**Review by khushalarora — CHANGES_REQUESTED** [review]
Hi @qadatta, thank you for contributing  :tada: 
Please rename your file and add  **' data/ '** at the beginning :+1: 
It will look like  `data/qadatta.json`

**Comment by qadatta** [comment]
> 

Updated file path to data/qadatta.json

**Review by khushalarora — APPROVED** [review]
Thank you  :tada:

**Review by kumarsonsoff3 — APPROVED** [review]
Looks awesome. Great collaboration everyone. 👍

**Pull request merged**